README.md

```diff
@@ -314,6 +316,27 @@ Consult the [Kroki documentation](https://kroki.io/#support) to find out which f
 | `kroki-fetch-diagram` | Define if we should download (and save on the disk) the images from the Kroki server.<br/>This feature is not available when running in the browser. | `false`
 | `kroki-http-method` | Define how we should get the image from the Kroki server. Possible values:<br/><ul><li>`get`: always use GET requests</li><li>`post`: always use POST requests</li><li>`adaptive`: use a POST request if the URI length is longer than 4096 characters, otherwise use a GET request</li></ul> | `adaptive` |
 
+### Default configuration
+
+By default, images are generated as svg when possible.
+To alter this, set `kroki-default-format` as a global, header, or page attribute:
```

Comment:
It might be misleading, attributes can be assigned to a document:
- through the CLI or API
- in the document’s header
- in the document’s body

A "page attribute" is definitely an Antora concept, you probably meant document body attribute?

Reply:
I have a different wording now, WDYT?